src/server/session.ts

```diff
@@ -2408,6 +2408,7 @@ namespace ts.server {
                 kind: item.kind,
                 kindModifiers: item.kindModifiers,
                 file: item.file,
+                containerName: item.containerName,

```

Comment:
This line is the implementation fix. Other changes are fixes to test infrastructure that was never used before now.